recipe/meta.yaml

```diff
@@ -46,8 +46,10 @@ outputs:
       host:
         - python
         - pip
-        - cython >=0.29.21
+        - packaging  # [osx and arm64]
+        - cython >=0.29.24
         - setuptools <60.0.0
+        - wheel 0.37.0
```

Comment:
why pinnng to an exact version here?  Is >=0.37.0 unsuitable here?

Reply:
I think wheel >=0.37.0 can be suitable. I guess v0.37.0 (with python 3.10 support) was used for maintenance and we know that 1.21.x won't use python 3.11 consequently wheel 0.37.0 should be only with python 3.10 support and not other upper versions.

Reply:
I've fixed it:
- `wheel >=0.37.0`